app/src/main/java/com/bootcamp_android/parking_app/adapter/Provider.kt

```diff
@@ -1,6 +1,6 @@
 package com.bootcamp_android.parking_app.adapter
 
-import com.bootcamp_android.parking_app.Lot
+
 
 class Provider {
```

Comment:
Function name should not contain underscores

Reply:
ok